cmd/config/aws/create_role_cmd_test.go

```diff
@@ -0,0 +1,31 @@
+package aws
+
+import (
+	"github.com/armory/armory-cli/pkg/config"
+	clitesting "github.com/armory/armory-cli/pkg/testing"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func getStdTestConfig(token string, outFmt string) *config.Configuration {
+	addr := "https://localhost"
+	clientId := ""
+	clientSecret := ""
+	isTest := true
+	return config.New(&config.Input{
+		AccessToken:  &token,
+		ApiAddr:      &addr,
```

Comment:
I prefer doing `lo.ToPtr("https://localhost")` these days but it's up to you.

Reply:
Much better - agreed :)